components/nodemanager-service/pgdb/migration/sql/02_add_unique_to_tags.up.sql

```diff
@@ -1,5 +0,0 @@
-ALTER TABLE IF EXISTS tags
```

Comment:
Probably not worth blocking over but it'd probably be nicer to have this just commented out with a comment explaining what happened.